gtsfm/bundle/bundle_adjustment.py

```diff
@@ -199,7 +196,7 @@ def run(
             logger.error(
                 "Bundle adjustment aborting, optimization cannot be performed without any tracks or any cameras."
             )
-            return initial_data, GtsfmMetricsGroup(name=METRICS_GROUP, metrics=[])
+            return initial_data, initial_data
```

Comment:
below, a tuple of GtsfmData is returned, but here, we were erroneously returning a MetricsGroup. Only `evaluate()` should return a MetricsGroup.

Reply:
Can we rename both the variables to something else? Also, is there a use for unfiltered data?

Reply:
Thanks for the review @ayushbaid. I think renaming would just introduce a copy, so I'm not sure I see the benefit. But I can be convinced if you feel otherwise : - )